### DISCUSSION TITLE
aliases for well-known canisters

### DISCUSSION THREAD

**Discussion by hansl** [dfinity/sdk#2645] [Feature Requests]
So users could do `dfx canister --network ic call @ledger ...` where `@ledger` is an invalid project canister name, and we do any resolution ourselves. Could also be reused with DNS resolution.

**Comment by p-shahi** [comment]
https://www.notion.so/Cycle-Wallet-Supporting-Flags-in-DFX-9596b431d45f4aec909ed99c24d4ab42
